tests/models/kosmos2_5/test_modeling_kosmos2_5.py

```diff
@@ -0,0 +1,770 @@
+            # self.assertTrue(embeddings.weight.shape, decoding.weight.shape)
+            # self.assertTrue(check_same_values(embeddings, decoding))
+
+            # # Check that after modification, they remain the same.
+            # decoding.weight.data.div_(4)
+            # # Check that the embedding layer and decoding layer are the same in size and in value
+            # self.assertTrue(embeddings.weight.shape, decoding.weight.shape)
+            # self.assertTrue(check_same_values(embeddings, decoding))
+
+            # Check that after resize they remain tied.
+            model_tied.resize_token_embeddings(config.text_config.vocab_size + 10)
+            params_tied_2 = list(model_tied.parameters())
+            self.assertEqual(len(params_tied_2), len(params_tied))
+
+            # decoding.weight.data.mul_(20)
+            # # Check that the embedding layer and decoding layer are the same in size and in value
+            # self.assertTrue(model.transformer.wte.weight.shape, model.lm_head.weight.shape)
+            # self.assertTrue(check_same_values(model.transformer.wte, model.lm_head))
+
+    @slow
+    def test_model_from_pretrained(self):
+        model_name = "microsoft/kosmos-2.5"
+        model = Kosmos2_5Model.from_pretrained(model_name)
+        self.assertIsNotNone(model)
+
+    @unittest.skip(reason="Does not work on the tiny model as we keep hitting edge cases.")
+    def test_model_parallelism(self):
+        super().test_model_parallelism()
+
+    # TODO: ydshieh
+    @require_torch_gpu
+    @pytest.mark.flash_attn_test
+    @slow
+    @unittest.skip(reason="kosmos-2.5 flash attention does not support right padding")
+    def test_flash_attn_2_inference_equivalence_right_padding(self):
+        pass
+
+    # TODO: ydshieh
+    @require_torch_gpu
+    @pytest.mark.flash_attn_test
+    @slow
+    @unittest.skip(reason="kosmos-2.5 test : the dummy inputs should be tweaked: dummy_input = inputs_dict")
+    def test_flash_attn_2_inference_equivalence(self):
+        pass
+
+    # TODO: ydshieh
+    @require_torch_sdpa
+    @require_torch_gpu
+    @slow
+    @unittest.skip(reason="_update_causal_mask is not implemented yet which fails this test")
+    def test_sdpa_can_dispatch_on_flash(self):
+        pass
+
+    # TODO: ydshieh
+    @unittest.skip(reason="doesn't support padding yet")
+    def test_eager_matches_sdpa_inference_1_bfloat16(self):
+        pass
+
+    # TODO: ydshieh
+    @unittest.skip(reason=" the model hasn't been added to auto class")
+    def test_flash_attn_2_from_config(self):
+        pass
+
+    @unittest.skip("This test is currently not well designed for multimodal model (float type as an input).")
+    def test_flash_attn_2_fp32_ln(self):
+        pass
+
+    @unittest.skip("This test is currently not well designed for multimodal model (float type as an input).")
+    def test_flash_attention_2_padding_matches_padding_free_with_position_ids(self):
+        pass
+
```

Comment:
You should add the following skipped test as well to avoid the test from failing:
```
@unittest.skip("Kosmos 2.5 is multimodel and has specific input shapes.")
    def test_flash_attn_2_generate_reuse_cache(self):
        pass
```